Client/WarFare/PlayerMySelf.cpp

```diff
@@ -829,7 +836,7 @@ bool CPlayerMySelf::CheckCollision()
 			if(fMag < fHeightSum) // �Ÿ��� Ű�� �պ��� �����..
 			{
 				float fMag2 = (pUPC->Position() - vPosNext).Magnitude(); // ����ġ�� �� ������.
-				if(fMag2 < fMag)
+				if(s_pPlayer->m_InfoBase.iAuthority != AUTHORITY_MANAGER && fMag2 < fMag)
```

Comment:
In the case of the above check, this player and below NPC check, and the shape collision check, could we not simply wrap these all in one conditional to exclude them from being handled in their entirety as a GM?

Those don't set anything.

Reply:
Exactly what i though which is why i commented descriptive info on this function. But since it updates the next y coordinates while doing collision checks, i haven't really had the time to refactor this.